lms/static/scripts/frontend_apps/components/dashboard/OrganizationActivity.tsx

```diff
@@ -25,11 +21,13 @@ const courseURL = (id: number) => urlPath`/courses/${String(id)}`;
 /**
  * List of courses that belong to a specific organization
  */
-export default function OrganizationActivity({
-  organizationPublicId,
-}: OrganizationActivityProps) {
+export default function OrganizationActivity() {
   const { dashboard } = useConfig(['dashboard']);
   const { routes } = dashboard;
+  const { organizationPublicId } = useParams<{
+    organizationPublicId: string;
+  }>();
```

Comment:
Hmm... what does this do if nested routes try to use parameters with the same name? Overwrite them with the value from the innermost route? I don't expect this to be a problem in any case since we always design route names to have unique names for each parameter.

Reply:
> what does this do if nested routes try to use parameters with the same name? Overwrite them with the value from the innermost route?

There's a note in [the example here](https://github.com/molefrog/wouter/releases/tag/v3.3.0) which says it does precisely that.